specification/maps/data-plane/Microsoft.Maps/Dataset/preview/1.0/examples/Dataset_Create.json

```diff
@@ -8,19 +8,19 @@
   "responses": {
     "201": {
       "headers": {
-        "Location": "https://atlas.microsoft.com/dataset/{datasetId}?api-version=1.0",
```

Comment:
If you do the parameterized host thing, then all the examples will also have to add
"geography": "us",

as a parameter